src/atomate2/common/schemas/phonons.py

```diff
@@ -514,6 +515,8 @@ def from_forces_born(
             },
         )
 
+        return doc.model_copy(update=additional_fields)
```

Comment:
Just wondering if that isn't very slow. Is there no other way to update the document?

Reply:
I will check again. I did not find any other way to do it so far . I also had looked in emmet for vasp doc 

https://github.com/materialsproject/emmet/blob/fdbcae26e0a00c22186bfdb2899c4cc929ba7125/emmet-core/emmet/core/tasks.py#L604.

Reply:
I see